src/NumericalAlgorithms/Integration/GslQuadAdaptive.hpp

```diff
@@ -7,28 +7,25 @@
 #include <gsl/gsl_integration.h>
 #include <tuple>
 
-#include "Parallel/Printf.hpp"
-
 /// \cond
 namespace PUP {
 class er;
 }  // namespace PUP
 /// \endcond
 
+/// Implements adaptive integration algorithms from GSL.
```

Comment:
the namespace will probably be used for non-GSL functions as well, so please remove the reference to GSL. Perhaps just `Numerical integration algorithms`